### DISCUSSION TITLE
how to export all milvus records to other database like hbase?

### DISCUSSION THREAD

**Discussion by buptzouy** [milvus-io/milvus#17993] [Q&A and General discussion]
Is there any api to support the operation?

**Comment by yhmo** [comment]
Milvus stores data files in MinIO, stores metadata in etcd, and un-flushed data in Pulsar. It is hard to export the data in a simple way, we intend to provide data backup tool. But API to export data to the original format may be a long-term plan.

**Reply by buptzouy** [reply]
Thanks
